packages/react-components/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -123,35 +123,35 @@ const useRootStyles = makeStyles({
   },
   subtle: {
     backgroundColor: tokens.colorSubtleBackground,
-    ...shorthands.borderColor('transparent'),
+    ...shorthands.borderColor(tokens.colorTransparentStroke),
     color: tokens.colorNeutralForeground2,
 
     ':hover': {
       backgroundColor: tokens.colorSubtleBackgroundHover,
-      ...shorthands.borderColor('transparent'),
-      color: tokens.colorNeutralForeground2BrandHover,
+      ...shorthands.borderColor(tokens.colorTransparentStroke),
+      color: tokens.colorNeutralForeground2,
```

Comment:
I haven't looked at the actual colors but it seems like this should be `colorNeutralForeground2Hover`?

Reply:
The figma file specifies it like this but I have asked Daisy and I'm waiting for her response since it did seem weird to me as well.

Reply:
This has been changed in both the design docs and here!